Evaluation_1/TFHub_Sample_Deploy/export.py

```diff
@@ -0,0 +1,66 @@
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import tensorflow_hub as hub
+""" Create a Sample TF-Hub Module using SavedModel v2.0
+The module has as a single signature which loads MNIST Dataset from TFDS and train a simple Neural Network for classifying the digits. The model is built and trained using Tewnsorlfow 
+"""
+
+class MNIST(tf.keras.models.Model):
```

Comment:
Add doctrings to classes & functions. If a lot is in flux, file an Issue on the repo to do this later.

Reply:
Created an Issue for it :)